
costmap:
  global_frame: base_footprint
  robot_base_frame: base_footprint
  footprint: [[-0.475, -0.355], [-0.475, 0.355], [0.201, 0.355], [0.201, -0.355]]
  footprint_padding: 0.085  
  rolling_window: true
  height: 10
  origin_x: 0.0
  origin_y: 0.0
  resolution: 0.05
  robot_radius: 0.46
  width: 10
  update_frequency: 20.0
  publish_frequency: 20.0
  always_send_full_costmap: true
  transform_tolerance: 2.0  
  inflation_layer: {cost_scaling_factor: 5.0, enabled: true, inflation_radius: 0.3}
  obstacle_layer:
    combination_method: 1
    enabled: true
    inflation_radius: 0.0
    laser_scan_sensor: {clearing: true, data_type: LaserScan, marking: true, topic: scan}
    max_obstacle_height: 0.5
    min_obstacle_height: 0.1
    observation_sources: laser_scan_sensor
    obstacle_range: 10.0
    raytrace_range: 11.0
    track_unknown_space: false
  rtobstacle_layer:
    combination_method: 1
    enabled: true
    inflation_radius: 0.0
    laser_scan_sensor: {clearing: true, data_type: LaserScan, marking: true, topic: scan, expected_update_rate: 0.15}
    max_obstacle_height: 0.5
    min_obstacle_height: 0.1
    observation_sources: laser_scan_sensor
    track_unknown_space: false
    footprint_clearing_enabled: false
  plugins:
    - {name: rtobstacle_layer, type: 'rt_obstacle_layer::RtObstacleLayer'}
    - {name: inflation_layer, type: 'costmap_2d::InflationLayer'}
